sys/shell/commands/sc_icmpv6_echo.c

```diff
@@ -79,11 +79,12 @@ int _handle_reply(ng_pktsnip_t *pkt, uint64_t time)
 
     if ((byteorder_ntohs(icmpv6_hdr->id) == id) &&
         (byteorder_ntohs(icmpv6_hdr->seq) == seq)) {
-        printf("%zu bytes from %s: id=%" PRIu16 " seq=%" PRIu16 " hop limit=%" PRIu8
-               "time = %" PRIu64 ".%03" PRIu64 " ms\n", icmpv6->size,
+        timex_t rt = timex_from_uint64(time);
+        printf("%u bytes from %s: id=%" PRIu16 " seq=%" PRIu16 " hop limit=%" PRIu8
+               " time = %" PRIu32 ".%03" PRIu32 " ms\n", icmpv6->size,
```

Comment:
`icmpv6->size` needs cast to `unsigned int` then, otherwise there will be compiler warnings.